### PR TITLE
Support YAML output #18

### LINKED ISSUE #18: Support YAML output
Support results output in YAML format, as an example:

```json
---
  movies:
    - primary title: ...
      original title: ...
      ...
    - primary title: ...
      ...
  series:
    - primary title: ...
      ...
```

### PR DESCRIPTION
**Pull request by Olsi-B**
Closes #18.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -30,6 +30,7 @@ enum-utils = "0.1"
 serde = { version = "1.0", features = ["derive"] }
 # serde_with = "1.9"
 serde_json = "1.0"
+serde_yaml = "0.8.23"
```
Comment (fredmorcos) [inline Cargo.toml:33]: You only need `0.8` here so that we get minor version updates automatically with `cargo update`.

**Review by fredmorcos — CHANGES_REQUESTED** [review]
Awesome! Thank you so much!

One minor change and we can merge it!

**Comment by fredmorcos** [comment]
Oh. I almost forgot. Can you also add documentation about using the `--output` flag to the `README`?

**Review by fredmorcos — APPROVED** [review]

**Comment by fredmorcos** [comment]
Awesome! I'll prepare `0.7.2` in a few hours.

**Comment by Olsi-B** [comment]
Great.

**Pull request merged**